src/task_4_2.py

```diff
@@ -0,0 +1,19 @@
+# Дан список целых чисел. Подсчитать сколько четных чисел в списке
+
+list_1, rez, a = [1, 2, 3, 4, 5, 6], 0, 0
+
+# ------------------------WHILE----------------------------------------
+
+while a < len(list_1):
+    if list_1[a] % 2 == 0:
+        rez += 1
+    a += 1
+print(rez)
+
+# -------------------------FOR-----------------------------------------
+
+rez = 0
+for i in list_1:
+    if list_1[i - 1] % 2 == 0:
```

Comment:
код не будет работать верно если в списке будут числа [2, 1, 2, 2, 2, 2]

проверять на цело тебе нужно именно i
```suggestion
    if list_1[i - 1] % 2 == 0:
```